monkey/monkey_island/cc/ui/src/components/pages/ReportPage.js

```diff
@@ -297,20 +298,24 @@ class ReportPageComponent extends AuthComponent {
               return x === true;
             }).length > 0 ?
               <div>
-                During this simulated attack the Monkey uncovered <span
+                During this simulated attack the Monkey uncovered  <span
                 className="label label-warning">
                     {this.state.report.overview.issues.filter(function (x) {
                       return x === true;
-                    }).length} threats</span>:
+                    }).length} threats.</span>:
```

Comment:
This will look like this: `During this simulated attack the Monkey uncovered 9 threats.:`  Remove the dot

Reply:
Removed the `.`